Functions/xASL_qc_CollectQC_Structural.m

```diff
@@ -35,10 +35,13 @@
     Struct.ID = x.SUBJECTS{iSubject};
 
     PathOrientationResults = fullfile(x.dir.SUBJECTDIR,'xASL_qc_PrintOrientation_RigidRegT1.tsv');
-    TempS = xASL_im_DetermineFlip(x, iSubject, PathOrientationResults, Struct);
-    Struct.T1w_LR_flip_YesNo = TempS.LR_flip_YesNo;
+    Struct.T1w_LR_flip_YesNo = uint8(xASL_im_DetermineFlip(PathOrientationResults));
```

Comment:
Same here. We could use `x.modules.structural.T1w_LR_flip_YesNo`, this way you can still find it in the resulting x struct 😃

Reply:
Ah sorry, I saw that this `ASL` struct is stored in x.Output later on. Then my request doesn't really make sense 👍 I'll mark it as resolved.